app/src/main/java/org/projectbuendia/client/ui/chart/ChartRenderer.java

```diff
@@ -149,6 +149,12 @@ public void render(Chart chart, Map<String, Obs> latestObservations,
                 mTileRows.add(tileRow);
             }
             for (ChartSection section : chart.rowGroups) {
+                // To create a row for the chart section we have to add any UUID.
```

Comment:
Suggest "we have to add an arbitrary UUID".

Is it possible to set this arbitrary UUID to `null` or `""` instead of the concept for notes?